<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuWindow</class>
 <widget class="QMainWindow" name="MenuWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MenuWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-21</y>
      <width>1091</width>
      <height>721</height>
     </rect>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="background">
      <property name="geometry">
       <rect>
        <x>-50</x>
        <y>-20</y>
        <width>1200</width>
        <height>750</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="Resources.qrc">:/img/img/main_bg_2.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="play_btn">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>450</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #F8E9EC;
background-color: #90013B;
text-transform: uppercase;</string>
      </property>
      <property name="text">
       <string>Play</string>
      </property>
     </widget>
     <widget class="QLabel" name="img_kirby">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>270</y>
        <width>175</width>
        <height>175</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="Resources.qrc">:/img/img/main_kirby_5.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="img_scarfy">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>420</y>
        <width>150</width>
        <height>150</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="Resources.qrc">:/img/img/main_scarfy.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="img_bomber">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>500</y>
        <width>175</width>
        <height>175</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="Resources.qrc">:/img/img/main_bomber.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="img_dee">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>450</y>
        <width>225</width>
        <height>225</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="Resources.qrc">:/img/img/main_dee.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="title">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>70</y>
        <width>321</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="Resources.qrc">:/img/img/main_title.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="subtitle">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>170</y>
        <width>251</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="Resources.qrc">:/img/img/main_subtitle.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>544</y>
        <width>381</width>
        <height>101</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="difficulty_container">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #F8E9EC;
background-color: #E54F79;
text-transform: uppercase;</string>
         </property>
         <property name="text">
          <string>Difficulty</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="easy_btn">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
padding: 20px 0;
text-transform: uppercase;
color: #F8E9EC;
background-color: #90013B;
}
</string>
           </property>
           <property name="text">
            <string>Easy</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="medium_btn">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
padding: 20px 0;
text-transform: uppercase;
color: #90013B;
background-color: #F8E9EC;
}

QPushButton:hover {
padding: 20px 0;
text-transform: uppercase;
color: #F8E9EC;
background-color: #90013B;
}
</string>
           </property>
           <property name="text">
            <string>Medium</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="hard_btn">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
padding: 20px 0;
text-transform: uppercase;
color: #90013B;
background-color: #F8E9EC;
}

QPushButton:hover {
padding: 20px 0;
text-transform: uppercase;
color: #F8E9EC;
background-color: #90013B;
}
</string>
           </property>
           <property name="text">
            <string>Hard</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="custom_btn">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
padding: 20px 0;
text-transform: uppercase;
color: #90013B;
background-color: #F8E9EC;
}

QPushButton:hover {
padding: 20px 0;
text-transform: uppercase;
color: #F8E9EC;
background-color: #90013B;
}
</string>
           </property>
           <property name="text">
            <string>Custom</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>330</y>
        <width>441</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="arrow_container">
       <property name="spacing">
        <number>300</number>
       </property>
       <item>
        <widget class="QPushButton" name="left_arrow">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: transparent; 
border: 0px;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="Resources.qrc">
           <normaloff>:/img/img/arrow_left.png</normaloff>:/img/img/arrow_left.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="right_arrow">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: transparent; 
border: 0px;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="Resources.qrc">
           <normaloff>:/img/img/arrow_right.png</normaloff>:/img/img/arrow_right.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>20</y>
        <width>141</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="music_btn">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid #E54F79;
background: #F8E9EC;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="Resources.qrc">
           <normaloff>:/img/img/music_kirby.png</normaloff>:/img/img/music_kirby.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="custom_btn_2">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid #E54F79;
background: #F8E9EC;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="Resources.qrc">
           <normaloff>:/img/img/tutorial_kirby.png</normaloff>:/img/img/tutorial_kirby.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <zorder>background</zorder>
     <zorder>play_btn</zorder>
     <zorder>img_scarfy</zorder>
     <zorder>img_bomber</zorder>
     <zorder>img_dee</zorder>
     <zorder>title</zorder>
     <zorder>subtitle</zorder>
     <zorder>verticalLayoutWidget</zorder>
     <zorder>horizontalLayoutWidget_2</zorder>
     <zorder>img_kirby</zorder>
     <zorder>horizontalLayoutWidget</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
